tests/test_scc.py

```diff
@@ -50,7 +50,7 @@ def test_empty_file(self, sample_scc_empty):
             SCCReader().read(sample_scc_empty)
 
     def test_positioning(self, sample_scc_multiple_positioning):
-        captions = SCCReader().read(sample_scc_multiple_positioning)
+        captions = SCCReader().read(sample_scc_multiple_positioning, src_fps=25.0)
```

Comment:
This test doesn't check the timestamps and it's only focused on the SCCReader, please write dedicated tests for both SCCReader and SCCWriter and check if the resulted timestamps are correct.